payloads/v10/channel.ts

```diff
@@ -2064,7 +2064,10 @@ export enum ChannelFlags {
 /**
  * https://discord.com/developers/docs/interactions/message-components#message-components
  */
-export type APIMessageComponent = APIMessageActionRowComponent | APIMessageTopLevelComponent | APIThumbnailComponent;
+export type APIMessageComponent =
+	| APIMessageActionRowComponent
+	| APIMessageTopLevelComponent
+	| APISectionAccessoryComponent;
```

Comment:
Really? Buttons work too?

Reply:
well buttons are already in APIMessageActionRowComponent - I think this type is just any component that can be anywhere inside a message